src/tests/ftest/launch.py

```diff
@@ -762,6 +762,7 @@ def run_tests(test_files, tag_filter, args):
         "--ignore-missing-references on",
         "--show-job-log" if not args.sparse else "",
         "--html-job-result on",
+        "--sysinfo on",
```

Comment:
This is no enough and I believe the default state of sysinfo collection.

What you need to do is add collection of `dmesg` as a _post_ operation to the sysinfo collection.